README.md

```diff
@@ -31,7 +31,7 @@ Please refer to [Readme.md](https://github.com/Azure/service-fabric/blob/master/
 ## Releases and Support
 Official releases from Microsoft of the NuGet packages in this repo are released directly to NuGet and Web Platform Installer. Get the latest official release [here](http://www.microsoft.com/web/handlers/webpi.ashx?command=getinstallerredirect&appid=MicrosoftAzure-ServiceFabric-VS2015).
 

```

Comment:
Release link above doesn't seem to work. Also we should replace above link by the one for VS 2017.

Reply:
Thanks, Link is working, we only release VS 2015 tooling in our SDK, VS 2017 SF tools are part of VS 2017 release.